src/Components/dashboard/EditEvents.js

```diff
@@ -1,5 +1,203 @@
+                label="Date"
+                className={classes.textFieldStyle}
+                value={dateEvent}
+                onChange={(date) => {
+                  setDateEvent(dayjs(date));
+                }}
+                format={dateFormats.customDate}
+                sx={{
+                  borderBottom: "1px solid rgb(179, 175, 177)",
+                }}
+                InputProps={{ disableUnderline: false }}
+                InputLabelProps={{ className: "text-label" }}
+              />
+            </LocalizationProvider>
+            <TextField
+              label="Time"
+              type="text"
+              variant="standard"
+              className={classes.textFieldStyle}
+              sx={{
+                "& .MuiInputLabel-root": {},
+                borderBottom: "1px solid rgb(179, 175, 177)",
+              }}
+              InputProps={{ disableUnderline: true }}
+              InputLabelProps={{ className: "text-label" }}
+              onChange={(e) => {
+                setTimeEvent(e.target.value);
+              }}
+              value={timeEvent}
+            ></TextField>
+            <TextField
+              label="Title"
+              type="text"
+              variant="standard"
+              className={classes.textFieldStyle}
+              sx={{
+                "& .MuiInputLabel-root": {},
+                borderBottom: "1px solid rgb(179, 175, 177)",
+              }}
+              InputProps={{ disableUnderline: true }}
+              InputLabelProps={{ className: "text-label" }}
+              onChange={(e) => {
+                setTitleEvent(e.target.value);
+              }}
+              value={titleEvent}
+            ></TextField>
+            <TextField
+              label="Description"
+              type="text"
+              variant="standard"
+              className={classes.textFieldStyle}
+              sx={{
+                "& .MuiInputLabel-root": {},
+                borderBottom: "1px solid rgb(179, 175, 177)",
+              }}
+              InputProps={{ disableUnderline: true }}
+              InputLabelProps={{ className: "text-label" }}
+              onChange={(e) => {
+                setDescriptionEvent(e.target.value);
+              }}
+              value={descriptionEvent}
+            ></TextField>
+            <TextField
+              label="Capacity"
+              type="text"
+              variant="standard"
+              className={classes.textFieldStyle}
+              sx={{
+                "& .MuiInputLabel-root": {},
+                borderBottom: "1px solid rgb(179, 175, 177)",
+              }}
+              InputProps={{ disableUnderline: true }}
+              InputLabelProps={{ className: "text-label" }}
+              onChange={(e) => {
+                setCapacityPeopleEvent(e.target.value);
+              }}
+              value={capacityPeopleEvent}
+            ></TextField>
+          </div>
+        </div>
```

Comment:
Instead of having all these code, you can just create another component call `EditEventCard.js`